sdk/src/AbstractMain.cpp

```diff
@@ -230,14 +232,17 @@ int AbstractMain::run(int in_argc, char** in_argv)
 
    if (options.getLogLevel() > LogLevel::INFO)
```

Comment:
Let's remove this if statement. We will always want to use a file log destination now, as opposed to using a SysLogDestination which we did previously. This is to fit the new logging paradigm we are using organization-wide.